docs/intro/contributing.txt

```diff
@@ -82,8 +89,14 @@ development version of Django and to generate patch files for the changes you
 make.
 
 To check whether or not you have Git installed, enter ``git`` into the command
-line. If you get messages saying that this command could not be found, you'll have
-to download and install it, see `Git's download page`__.
+line. If you get messages saying that this command could not be found, you'll
+have to download and install it, see `Git's download page`__.
+
+.. note::
```

Comment:
`.. admonition:: For Windows users` will give a more helpful title in the rendered docs.